core/src/display_object/movie_clip.rs

```diff
@@ -1276,10 +1276,18 @@ impl<'gc> MovieClip<'gc> {
         self.0.write(gc_context).current_frame = current_frame;
     }
 
-    pub fn frames_loaded(self) -> i32 {
+    // Gets the actual number of frames loaded (ignoring the SWF header)
+    pub fn real_frames_loaded(self) -> i32 {
         self.0.read().frames_loaded()
     }
 
+    // Gets the number of frames loaded, clamping it to the number of frames
+    // declared in the SWF header. This is the value we report to AVM code
+    // to match Flash Player's behavior
+    pub fn frames_loaded_for_avm(self) -> i32 {
```

Comment:
can we rename these to be more consistent? As in, since both `frames_loaded_for_avm` and `total_frames` are based on what the header says, I'd say both should have the same naming scheme - either no suffix, or `header_` (for example).
`frames_loaded_for_avm` doesn't seem to fit, as it's used outside of pure AVM code.